core/constant_inc.php

```diff
@@ -577,7 +577,7 @@
 define( 'EMAIL_SHUTDOWN_FORCE', 2 );
 
 # Lengths - NOTE: these may represent hard-coded values in db schema and should not be changed.
-define( 'DB_FIELD_SIZE_USERNAME', 255 );
+define( 'DB_FIELD_SIZE_USERNAME', 191 );
 define( 'DB_FIELD_SIZE_REALNAME', 255 );
```

Comment:
There is no index on realname.
The length of the index is limited.
https://www.mantisbt.org/bugs/view.php?id=20431#c52209